api/tests/core/search/test_serialize_algolia.py

```diff
@@ -216,6 +220,19 @@ def test_serialize_offer_thumb_url():
     assert serialized["offer"]["thumbUrl"] == f"/storage/thumbs/products/{humanize(offer.productId)}"
 
 
+def test_serialize_offer_gtl():
+    product = offers_factories.ProductFactory(extraData={"gtl": "01030100"})
+    offer = offers_factories.OfferFactory(
+        product=product,
+        subcategoryId=subcategories.LIVRE_PAPIER.id,
+    )
+    serialized = algolia.AlgoliaBackend().serialize_offer(offer, 0)
+    assert serialized["offer"]["GTL_level1"] == "Littérature"
+    assert serialized["offer"]["GTL_level2"] == "Œuvres classiques"
+    assert serialized["offer"]["GTL_level3"] == "Antiquité"
+    assert serialized["offer"]["GTL_level4"] == None
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏
```suggestion
    assert serialized["offer"]["GTL_level4"] is None